src/main/kotlin/com/vaadin/gradle/VaadinBuildFrontendTask.kt

```diff
@@ -18,12 +18,11 @@ package com.vaadin.gradle
 import com.vaadin.flow.server.Constants
 import com.vaadin.flow.server.frontend.FrontendUtils
 import com.vaadin.flow.server.frontend.NodeTasks
-import elemental.json.JsonObject
-import elemental.json.impl.JsonUtil
 import org.gradle.api.DefaultTask
 import org.gradle.api.tasks.TaskAction
-import org.gradle.api.tasks.compile.AbstractCompile
+import org.gradle.api.tasks.bundling.Jar
 import java.io.File
+import java.nio.file.Files
```

Comment:
Is the Files import needed. Can't see it used and the next PR doesn't import it either,

Reply:
Good catch, removed.